src/js/iov/Player/IovPlayerCollection.js

```diff
@@ -176,6 +177,12 @@ export default class IovPlayerCollection extends EventEmitter {
       }
     });
 
+    iovPlayer.on(IovPlayer.events.JWT_AUTHORIZATION_FAILURE, async (data) => {
+      this.logger.error(`Critical JWT authorization error: ${data.error}`);
+      // prevent any retries
+      this.retryAttempts[id] = this.MAX_RETRIES_ON_PLAY_ERROR + 1;
+    });
+
```

Comment:
QUESTION: does this function need to be `async`? i don't see any `await` in it.

Reply:
I suppose not.  no need for async if there's no await.